pkg/initialise/initialise.go

```diff
@@ -0,0 +1,129 @@
+package initialise
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/api"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/download/yamlcreator"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/util"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/util/files"
+)
+
+//CreateTemplate Creates a blank set of monaco folders and files for each supported API
+func CreateTemplate(workingDir string, createSpecificAPI string) error {
+
+	util.Log.Info("Creating Config for APIs: %v", createSpecificAPI)
+
+	apiList, getAPIListErr := getAPIList(createSpecificAPI)
+	if getAPIListErr != nil {
+		return getAPIListErr
+	}
+
+	workingDir = filepath.Clean(workingDir)
+	util.Log.Info("Initialising Monaco Demo Folders")
+	projectsFolder := filepath.Join(workingDir, "projects")
+	demoProjectFolder := filepath.Join(workingDir, "projects", "baseconfig")
```

Comment:
It might make sense to provide the initial project name also via an command
line argument.

Reply:
I disagree. In this case, it's probably best to keep the commands as simple as possible. The folder can (and probably should) be renamed after the dummy content.

Reply:
I would take it as an optional argument though. So by default call it `baseconfig`
but when i provide param `n` (or whatever) it i want to override it.